src/components/Input/Input.module.css

```diff
@@ -4,29 +4,17 @@
   border: var(--rvr-border-width-md) solid var(--rvr-gray-60);
   color: var(--rvr-gray-90);
   font-family: var(--rvr-base-font-family);
-  font-size: var(--rvr-font-size-md);
   line-height: var(--rvr-line-height-lg);
+  outline: none;
```

Comment:
Since we provide our own `focus` styles, we don't want the browser's default in this case.

Reply:
Maybe this was due to some of the re-arranging I was doing in response to suggestions from `stylelint`. I'll revisit this on Monday